style.md

```diff
@@ -26,7 +26,7 @@ The exceptions are entries where the Go default for the type is a no-op in the s
 ### Anchoring
 
 For any given section that provides a notable example, it is ideal to have it denoted with [markdown headers][markdown-headers].
-The level of header should be such that it is a subheader of the header it is an example of.
+The level of header for an example is ideally such that it is a subheader of the header it is an example of.
```

Comment:
This sentence was a bit twisty before.  I think “should” vs. “SHOULD” is sufficiently distinct, but if folks want to drop non-SHOULD instances, we could maybe use an imperative like:

> Make example headers a child (not a deeper ancestor) of the header they illustrate.